src/appleseed-max-impl/appleseedinteractive/appleseedinteractive.cpp

```diff
@@ -220,8 +222,92 @@ namespace
                     break;
                 }
             }
+
+            //TODO: update lights here
+        }
+        
+        void MaterialStructured(NodeKeyTab& nodes)
+        {
+            if (m_renderer == nullptr)
```

Comment:
Is `m_renderer == nullptr` something that can normally happen, or would it be a programmer error? If the former, keep the code as is. If the latter, replace by an assertion.